src/test/java/de/tum/in/test/integration/testuser/subject/structural/SomeAbstractClass.java

```diff
@@ -14,4 +14,8 @@ protected SomeAbstractClass(String someString, int someInt) {
 	}
 
 	abstract void doNothing();
+
+	protected void nonAbstractProtected() {
+		// nothing
+	}
```

Comment:
Is this currently used?

It would be nice if we had tests that check the new recursive declared... search. In this case, we could check that this is found by searching for it starting in the `AbstractClassExtension`, and that this is not possible with the old, non-private variant.

Reply:
I wasn’t finished adding tests. It is used now by the additional tests.